deps/repl.lua

```diff
@@ -49,7 +49,7 @@ setmetatable(exports, {
   })
   global._G = global
 
-  if greeting then print(greeting) end
+  if greeting then stdout:write(greeting) end
```

Comment:
`print` adds a newline while `stdout:write` does not.  Doesn't this act weird when using the repl with normal stdout?